packages/edge/docs/interfaces/Geo.md

```diff
@@ -23,7 +24,7 @@ The city that the request originated from.
 
 #### Defined in
 
-[packages/edge/src/edge-headers.ts:47](https://github.com/vercel/vercel/blob/main/packages/edge/src/edge-headers.ts#L47)
+[packages/edge/src/edge-headers.ts:51](https://github.com/PaulieScanlon/vercel/blob/main/packages/edge/src/edge-headers.ts#L51)
```

Comment:
Can you fix these URLs as well?

Reply:
Ah man, I don't know how this happened. I think when I added the const declaration it auto-generated this stuff?

Reply:
yeah it's probably our script that did that. we might need to make `github.com/vercel` as a fixed url in the typedoc config! <picture data-single-emoji=":meow_sorry:" title=":meow_sorry:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/meow_sorry/8bcf9f1adf7c6c69.png" alt=":meow_sorry:" align="absmiddle"></picture>